DESCRIPTION

```diff
@@ -17,7 +17,9 @@ Imports:
     sf (>= 0.7.6),
     tmap (>= 2.3),
     mapdeck (>= 0.2.1)
-Remotes: mtennekes/tmap
+Remotes:
+    mtennekes/tmap,
+    SymbolixAU/mapdeck
```

Comment:
Got a few issues installing that first time inside the container. Let's see how it builds.

Reply:
Heads-up @mpadge I suggest adding this to the remotes:

```
    SymbolixAU/geojsonsf,
    SymbolixAU/spatialwidget,
    SymbolixAU/mapdeck
```

Reply:
We do not need tmap anymore.

Reply:
done that - thanks!